static/js/containers/App.js

```diff
@@ -91,6 +96,30 @@ class App extends React.Component {
     }
   }
 
+  updateProgramEnrollments() {
+    const { enrollments, dispatch } = this.props;
+    let programId = filterPositiveInt(window.localStorage.getItem("programId"));
+    if (
+      programId &&
+      enrollments.getStatus === FETCH_SUCCESS &&
+      enrollments.postStatus !== FETCH_PROCESSING &&
+      enrollments.postStatus !== FETCH_FAILURE
+    ) {
+      if ( !enrollments.programEnrollments.find(e => e.id === programId) ) {
+        dispatch(addProgramEnrollment(programId)).then(() => {
+          window.localStorage.removeItem("programId");
+        }, e => {
+          window.localStorage.removeItem("programId");
+          if ( e.errorStatusCode === 404 ) {
+            window.localStorage.removeItem("programId");
+          } else {
+            window.localStorage.setItem("programIdFailed", programId);
```

Comment:
I'm not sure if it makes sense to store this if we're not going to use it again. If we want to alert the user to a failure we should do a `console.error` instead or something more visible (cc @roberthouse54)